eos/utils/integrate.cc

```diff
@@ -20,14 +20,27 @@
 #include <eos/utils/integrate.hh>
 #include <eos/utils/matrix.hh>
 
+#include <gsl/gsl_errno.h>
+#include <gsl/gsl_integration.h>
+
 #include <limits>
 #include <vector>
 
+namespace
+{
+    double gsl_function_adapter(double x, void *params)
+    {
+        const auto& f = *static_cast<eos::GSL::fdd*>(params);
+        return f(x);
+    }
+}
+
 namespace eos
 {
     using std::abs;
     using std::real;
     using std::imag;
+    using namespace GSL;
```

Comment:
Why?

Reply:
A relict... after removing, I only had to prefix `GSL::fdd`